services/mailservice/mail.go

```diff
@@ -191,14 +190,14 @@ func TestConnection(config *model.Config) {
 
 	conn, err1 := ConnectToSMTPServer(config)
 	if err1 != nil {
-		mlog.Error(fmt.Sprintf("SMTP server settings do not appear to be configured properly err=%v details=%v", utils.T(err1.Message), err1.DetailedError))
+		mlog.Error("SMTP server settings do not appear to be configured properly", mlog.String("errorMessage", utils.T(err1.Message)), mlog.String("errorDetails", err1.DetailedError))
```

Comment:
Generally we have been using snake_case for the keys.

Reply:
Done